core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -611,12 +613,14 @@ class GroupMetadataManager(brokerId: Int,
                   case unknownKey =>
                     throw new IllegalStateException(s"Unexpected message key $unknownKey while loading offsets and group metadata")
                 }
+                bytesRead += record.sizeInBytes()
               }
             }
+            numMessagesRead += 1
```

Comment:
Looks like you're counting the number of batches, not the number of messages. Maybe both are useful.

Reply:
Sorry that was an oversight. I have corrected that plus added the batchesRead as well.

Reply:
@ijuma did you get a chance to look at the changes?